dashboard/config/locales/long_instructions.pt-BR.yml

```diff
@@ -9655,12 +10278,93 @@ pt-BR:
+      courseC_artist_loops1a_2019: |-
+        **Jogo Livre:** Vamos construir algo maior, um passo de cada vez!
+
+        Coloque o artista no centro da área de jogo usando o bloco <xml><block type="jump_to" block-text="jump to the middle center position"><title name="VALUE">13</title></block></xml>, então use o bloco <xml><block type="sticker" block-text="draw sticker"><title name="VALUE">Castor</title></block></xml> para fazer algo aparecer.
+
+        * * *
+
+        Pressione "Finalizar" quando estiver satisfeito com o que você fez.
+
+        ![expandable](https://images.code.org/1435d3650553df44eba973572d409a50-image-1553096405891.png)
+        ![expandable](https://images.code.org/6912d2144b90aa7087fe854742d1e8b9-image-1553096425394.png)
+        ![expandable](https://images.code.org/8de9e1aa15fdfe30ffed763e0e1d6e04-image-1553096428531.png)
+      courseC_artist_loops1b_2019: |-
+        Mude a direção pra qual seu adesivo está virado, adicionando um comando `girar` ao seu código.
+
+        * * *
+
+        Pressione "Finalizar" quando estiver satisfeito com o que você fez.
+
+        ![expandable](https://images.code.org/30fe9d37758abd27055d9a6e5a890680-image-1553097358155.png)
+        ![expandable](https://images.code.org/61feed00662019b99ac774322231f084-image-1553097362176.png)
+        ![expandable](https://images.code.org/fd123bb703ea4c630218a12eb5994053-image-1553097354066.png)
+      courseC_artist_loops2a_2019: |-
+        Agora use uma repetição para fazer mais cópias do seu adesivo. Tente mudar os números nos blocos `girar` e `repetir` para criar padrões diferentes.
+
+        * * *
+
+        Pressione "Finalizar" quando estiver satisfeito com o que você fez.
+
+        ![expandable](https://images.code.org/8b4352a1abe72419043d278752261b57-image-1553097917337.png)
+        ![expandable](https://images.code.org/5211e2fe2852e25a884a790155ba47d5-image-1553097923167.png)
+        ![expandable](https://images.code.org/0e3fbe8bbd20526cc735dfd26d2473ee-image-1553097925789.png)
+      courseC_artist_loops3a_2019: "Muita diversão!  \n\nAgora adicione um segundo
+        adesivo bem em cima do primeiro para criar uma imagem como as mostradas abaixo.\n\n*
+        * *\n\nPressione \"Finalizar\" quando estiver satisfeito com o que você fez.\n\n![expandable](https://images.code.org/3aa37f679eabc7a5a337291db614dd98-image-1496711069351.03.49.png)\n![expandable](https://images.code.org/dd0e0eee69d37493c5ba584164316a9f-image-1553098351969.png)\n![expandable](https://images.code.org/880908899d8e6ceac8a8ce1bd0bf47eb-image-1553098355311.png)"
+      courseC_artist_loops4a_2019: |-
+        Edite seu código para que você pule para frente entre os desenhos dos dois adesivos. Neste desafio, certifique-se de ter "pular para a posição central" dentro do seu laço de repetição, para que você fique no centro!
+
+        * * *
+
+        Pressione "Finalizar" quando você estiver feliz com o que fez.
+
+        [expandable] [0]
```

Comment:
This looks like a translation issue. I'll fix this in Crowdin but wasn't going to fix it in this PR.